app/main/forms/auth_forms.py

```diff
@@ -9,7 +10,9 @@ class LoginForm(Form):
         'Email address', id="input_email_address",
         validators=[
             DataRequired(message="You must provide an email address"),
-            Email(message="You must provide a valid email address")
+            Regexp("^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$",
```

Comment:
How about using the pattern that is used in the users json schema? 
`^[^@^\\s]+@[^@^\\.^\\s]+(\\.[^@^\\.^\\s]+)+$`

At least that way they can't use an email which will definitely fail later?